### PR TITLE
:tada: Add user feedback module.

### PR DESCRIPTION
**Pull request by niwinz**

### CODE REVIEW

File: backend/src/app/emails.clj [hunk 1]
```diff
@@ -43,6 +43,16 @@
...
 ;; --- Emails
 
+(s/def ::subject ::us/string)
+(s/def ::content ::us/string)
+
+(s/def ::feedback
+  (s/keys :req-un [::subject ::content]))
+
+(def feedback
+  "A new profile registration welcome email."
```
Comment (hirunatan) [inline backend/src/app/emails.clj]: This docstring should be updated, shouldn't it?

File: frontend/resources/styles/common/framework.scss [hunk 2]
```diff
@@ -20,6 +20,7 @@
   min-width: 25px;
   padding: 0 1rem;
   transition: all .4s;
+  text-decoration: none !important;
```
Comment (hirunatan) [inline frontend/resources/styles/common/framework.scss:23]: You said !important... HA-HA XD

File: backend/src/app/util/emails.clj [hunk 3]
```diff
@@ -196,15 +197,17 @@
         text (render-email-template-part :txt id context)
         html (render-email-template-part :html id context)]
     (when (or (not subj)
-              (not text)
-              (not html))
+              (or (not text)
```
Comment (hirunatan) [inline backend/src/app/util/emails.clj]: Shouldn't this be `(and`?

File: frontend/resources/styles/common/framework.scss (hunk 2 above)
Comment (niwinz) [reply frontend/resources/styles/common/framework.scss:23]: @elhombretecla stuff :P

**Pull request merged**